index.html

```diff
@@ -1,21 +1,52 @@
 <!DOCTYPE html>
 <html lang="pt">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Batalha Naval</title>
-    <script>
-      document.addEventListener("DOMContentLoaded", () => {
-        const username = localStorage.getItem("username");
-        if (!username) {
-          window.location.href = "/frontend/battleship/login/login.html";
-          return;
-        }
-        window.location.href = "/frontend/battleship/game.html";
-      });
-    </script>
-  </head>
-  <body>
-    <h1>Batalha Naval</h1>
-  </body>
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Batalha Naval</title>
+  <link rel="stylesheet" href="/Battleship/frontend/components/rules.css">
+  <link rel="stylesheet" href="/Battleship/frontend/components/ranking.css">
+  <link rel="stylesheet" href="/Battleship/frontend/css/style.css">
+</head>
+
+<body>
+  <div class="container">
+    <div class="left">
+      <h1 class="titulo-batalha">Batalha Naval</h1>
+      <p class="subtitulo">Destrua o seu oponente e conquiste os mares!</p>
+      
+      <div class="regras">
+        <iframe
+          src="/Battleship/frontend/components/rules.html"  
+          frameborder="0"
+          class="iframe-regras"
+          width="100%"
+          height="400px"  
+        ></iframe>
+      </div>
+
+      <!-- Botão para iniciar o jogo -->
+      <button class="play-btn" onclick="iniciarJogo()">
+        <span>Jogar</span>
+      </button>
+    </div>
+
+    <!-- Lado direito para o ranking -->
+    <div class="right">
+      <h2 class="titulo-ranking">Fique entre os 10 melhores jogadores!</h2>
+      <div class="ranking">
+        <iframe
+          src="/Battleship/frontend/components/ranking.html"
+          frameborder="0"
+          class="iframe-ranking"
+          width="100%"
+          height="400px" 
+        ></iframe>
+      </div>
+    </div>
+  </div>
+
+  <!-- Incluindo o arquivo JS -->
+  <script src="/Battleship/frontend/api/index.js"></script>
```

Comment:
```suggestion
  <script src="./frontend/api/index.js"></script>
```

Eu mudaria esse index.js pra a pasta de /scripts já que não tem nenhuma requisição para api do backend nesse arquivo js.